### ISSUE TITLE
SSM - get parameters by label doesn't work

### ISSUE THREAD

**Issue by John-D8** [localstack/localstack#3829]
# Type of request: This is a ...

[ x] bug report
[ ] feature request

# Detailed description

After adding a SSM parameter and labeling it, trying to get all parameters with the same label returns no parameters.

## Expected behavior

Parameter returned

## Actual behavior

No parameters returned

# Steps to reproduce

1) Add a parameter
2) Label the parameter version
3) Try to get parameters by path with the same label

## Command used to start LocalStack

docker-compose

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

```  
    path='my/path'
    value = 'value'
    res = ssm.put_parameter(Name=path, Value=value, Type='String')
    ssm.label_parameter_version(Name=path, ParameterVersion=res['Version'], Labels=['latest'])
    ps = ssm.get_parameters_by_path(Path=path,
                                    ParameterFilters=[
                                        {
                                            'Key': 'Label',
                                            'Values': [
                                                'latest',
                                            ]
                                        },
                                    ],
                                    )
  # returned ps contains empty array of parameters
```

**Comment by whummer** [comment]
Thanks for reporting @John-D8 . This should be fixed in #3854 - can you please pull the `latest` Docker image and give it another try? Please let us know if the problem persists. Thanks for your help!

**Comment by yonatandbinahai** [comment]
@whummer seems it still doesn't work in version `0.12.17.5`.
Can you please check?
(same scenario & result)
